IntegrationTestSuite/WebservicesIntegrationTests/Tests/SiteDirectory/POSTNewDomainExpertUser.json

```diff
@@ -529,6 +529,28 @@
       "objectClass": "ContractChangeNotice",
       "revisionNumber": 1
     },
+    //{
```

Comment:
i rather not have commented code in git, please remove comments or add a GH ticket to fix